public/src/js/models/article/images.js

```diff
@@ -3,7 +3,9 @@ export function thumbnail() {
         fields = this.fields,
         state = this.state;
 
-    if (meta.imageReplace() && meta.imageSrc()) {
+    if (meta.imageReplace() && meta.imageSrcThumb() && meta.imageSrcThumb() !== ''){
```

Comment:
minor: could create a `const` for `meta.imageSrcThumb()` to avoid multiple evaluations